### ISSUE TITLE
Misleading error when precompiled Erlang is not available

### ISSUE THREAD

**Issue by Nicd** [burrito-elixir/burrito#49]
I'm on macOS 11.6.5 (Big Sur) x86-64 and trying to build a Linux x86-64 release with burrito commit 8f25957ccb15073c876c1797a9315902effb05c5. I have Zig 0.9.1 installed, and I'm using Erlang 24.2.2, Elixir 1.13.4.

When I try to build my release, I get this error (after all the compiling of the deps):

```
Generated code_stats app
* assembling code_stats-2.3.0 on MIX_ENV=prod
* using config/runtime.exs to configure the release at runtime
* creating _build/prod/rel/code_stats/releases/2.3.0/vm.args
* creating _build/prod/rel/code_stats/releases/2.3.0/env.sh
* creating _build/prod/rel/code_stats/releases/2.3.0/env.bat
-> Burrito is building target: linux
-> Burrito will build for target:
        OS: linux
        CPU: x86_64
        Qualifiers: [libc: :glibc]
        Debug: false
--> PHASE: :fetch
---> Working directory: /var/folders/h9/b8n6qz715cj7ws460yq15v240000gn/T/burrito_build_C10476C909D9DDDE
---> Resolving ERTS: {:precompiled, [version: "24.2.2"]}
--> PHASE: :patch
** (CaseClauseError) no case clause matching: {:precompiled, [version: "24.2.2"]}
    (burrito 0.8.0) lib/steps/patch/copy_erts.ex:13: Burrito.Steps.Patch.CopyERTS.execute/1
    (burrito 0.8.0) lib/builder/builder.ex:146: anonymous fn/3 in Burrito.Builder.run_phase/2
    (elixir 1.13.3) lib/enum.ex:2396: Enum."-reduce/3-lists^foldl/2-0-"/3
    (elixir 1.13.3) lib/enum.ex:937: Enum."-each/2-lists^foreach/1-0-"/2
    (burrito 0.8.0) lib/builder/builder.ex:95: Burrito.Builder.build/1
    (mix 1.13.3) lib/mix/tasks/release.ex:1056: Mix.Tasks.Release.run_steps/1
    (mix 1.13.3) lib/mix/task.ex:397: anonymous fn/3 in Mix.Task.run_task/3
    (mix 1.13.3) lib/mix/cli.ex:84: Mix.CLI.run_task/2
```

I looked at the code and it seems there is no case clause for that, but I didn't look into it more yet to see how it all works. Please let me know if you need more info.

**Comment by Nicd** [comment]
Yep, changing version to 24.3.3 fixed the issue. I'll leave this up with a different title.

**Comment by Nicd** [comment]
I guess this is a duplicate of #39, just with more information. :)

**Comment by doawoo** [comment]
Thanks for the report! Agreed the error is misleading.

**Comment by doawoo** [comment]
Fixed in #50 :)
